x/pki/x509/x509.go

```diff
@@ -55,6 +71,24 @@ func DecodeX509Certificate(pemCertificate string) (*X509Certificate, error) {
 	return &certificate, nil
 }
 
+func formatOID(header []byte, oldKey []byte, newKey []byte) string {
```

Comment:
Better to receive arguments as `string` and make required conversion inside the function

Reply:
Done! ✅